quiz.js

```diff
@@ -0,0 +1,72 @@
+
+
+
+/*---- GET / TEST THE INPUT ------ */
+
+// function testInput(heightInput, charInput) {
+// 	console.log("It's working! ", heightInput, charInput);
+// 	console.log(input.height, input.character)
+// }
+
+function checkInput () {
+	var heightInput = document.getElementById("heightInput").value;
+	var charInput = document.getElementById("charInput").value;
+	
+	 if (heightInput && charInput){
+
+		beautifulTree(heightInput, charInput);
+	
```

Comment:
Note that the requirements state that you need to pass an object into your tree-growing function. So take these two variables and make them properties in an object instead. Then pass that object into your function. Come see me if you need help figuring that out